### ISSUE TITLE
Rule not triggered for MobileCore trackAction with custom name

### ISSUE THREAD

**Issue by emdobrin** [adobe/aepsdk-rulesengine-ios#30]
### Expected Behaviour
I should be able to trigger rules based on Mobile Core Track Action events with custom action name.

### Actual Behaviour
The rule is not triggered

#### Steps to Reproduce
Configure rule in Launch for Event Mobile Core, Track Action, Action equals test.actionName
and preferred consequence (e.g. postback, open url)
Send track action event on the client `MobileCore.track(action: "test.actionName", data: ["abc": "def"])`
The postback should be triggered, but it is not.

#### Sample Code that illustrates the problem

<details><summary>Sample rule</summary>
<p>
```
{
      "condition": {
        "type": "group",
        "definition": {
          "logic": "and",
          "conditions": [
            {
              "type": "group",
              "definition": {
                "logic": "or",
                "conditions": [
                  {
                    "type": "group",
                    "definition": {
                      "logic": "and",
                      "conditions": [
                        {
                          "type": "matcher",
                          "definition": {
                            "key": "~type",
                            "matcher": "eq",
                            "values": [
                              "com.adobe.eventType.generic.track"
                            ]
                          }
                        },
                        {
                          "type": "matcher",
                          "definition": {
                            "key": "~source",
                            "matcher": "eq",
                            "values": [
                              "com.adobe.eventSource.requestContent"
                            ]
                          }
                        },
                        {
                          "type": "matcher",
                          "definition": {
                            "key": "action",
                            "matcher": "eq",
                            "values": [
                              "test.actioname"
                            ]
                          }
                        }
                      ]
                    }
                  }
                ]
              }
            }
          ]
        }
      },
      "consequences": [
        {
          "id": "RC77054dce62a04369bfd5d6007700fde9",
          "type": "url",
          "detail": {
            "url": "https://www.example.com"
          }
        }
      ]
    }
```
</details>
<details><summary>Sample code</summary>
<p>
```
MobileCore.track(action: "test.actionName", data: ["abc": "def"])
```
</details>
<details><summary>Pods</summary>
<p>
```
PODS:
  - ACPCore (2.9.9):
    - AEPCore
    - AEPIdentity
    - AEPLifecycle
    - AEPRulesEngine
    - AEPServices
    - AEPSignal
  - AEPAssurance (1.0.0):
    - ACPCore
    - AEPAssurance/iOS (= 1.0.0)
  - AEPAssurance/iOS (1.0.0):
    - ACPCore
  - AEPCore (3.0.0-beta.1):
    - AEPRulesEngine
    - AEPServices
  - AEPEdge (1.0.0-beta-1):
    - AEPCore
    - AEPServices
  - AEPIdentity (3.0.0-beta.1):
    - AEPCore
  - AEPLifecycle (3.0.0-beta.1):
    - AEPCore
  - AEPRulesEngine (1.0.0-beta.1)
  - AEPSampleExtensionSwift (0.0.1):
    - AEPCore
  - AEPServices (3.0.0-beta.1)
  - AEPSignal (3.0.0-beta.1):
    - AEPCore
```
</details>

**Comment by shalehaha** [comment]
@emdobrin  It works for me. Do you have the log?
```
2020-11-02 21:49:18.860762-0700 SampleApp_Swift[2679:34397579] [AEP SDK TRACE - <EventHub:dispatch(event:)>] Event #15, id: 5A39F8C9-7928-4574-AA38-6C1BB850CF7C name: AnalyticsTrack type: com.adobe.eventType.generic.track source: com.adobe.eventSource.requestContent data: Optional(["action": "test.actionName"]) timestamp: 2020-11-03 04:49:18 +0000 responseId: nil is dispatched.
2020-11-02 21:49:18.866831-0700 SampleApp_Swift[2679:34399302] [AEP SDK TRACE - <Launch Rules Engine - LaunchRulesEngine>] Generating new consequence event id: 365F9EB0-95E6-432B-93ED-EA1E0B6F8D12 name: Rules Consequence Event type: com.adobe.eventType.rulesEngine source: com.adobe.eventSource.responseContent data: Optional(["triggeredconsequence": ["detail": ["templateurl": Optional("https://adobe.com"), "timeout": Optional(0)], "id": "RC20fcdaf852944ff9a23049e359e28eb3", "type": "pb"]]) timestamp: 2020-11-03 04:49:18 +0000 responseId: nil
2020-11-02 21:49:18.867090-0700 SampleApp_Swift[2679:34399302] [AEP SDK TRACE - <EventHub:dispatch(event:)>] Event #16, id: 365F9EB0-95E6-432B-93ED-EA1E0B6F8D12 name: Rules Consequence Event type: com.adobe.eventType.rulesEngine source: com.adobe.eventSource.responseContent data: Optional(["triggeredconsequence": ["detail": ["templateurl": Optional("https://adobe.com"), "timeout": Optional(0)], "id": "RC20fcdaf852944ff9a23049e359e28eb3", "type": "pb"]]) timestamp: 2020-11-03 04:49:18 +0000 responseId: nil is dispatched.
2020-11-02 21:49:18.874139-0700 SampleApp_Swift[2679:34398026] [AEP SDK TRACE - <SQLiteWrapper>] Successfully opened connection to database at /Users/jgeng/Library/Developer/CoreSimulator/Devices/C81B0AA6-15CE-49C1-BC48-47DFBDDCB7C1/data/Containers/Data/Application/B3A752D2-81D0-4DAA-9D32-235A975E2059/Library/Caches/com.adobe.module.signal
2020-11-02 21:49:18.877312-0700 SampleApp_Swift[2679:34399305] [AEP SDK TRACE - <SQLiteWrapper>] Successfully opened connection to database at /Users/jgeng/Library/Developer/CoreSimulator/Devices/C81B0AA6-15CE-49C1-BC48-47DFBDDCB7C1/data/Containers/Data/Application/B3A752D2-81D0-4DAA-9D32-235A975E2059/Library/Caches/com.adobe.module.signal
2020-11-02 21:49:18.878612-0700 SampleApp_Swift[2679:34399305] [AEP SDK DEBUG - <NetworkService>] Initiated (GET) network request to (https://adobe.com).
2020-11-02 21:49:19.433617-0700 SampleApp_Swift[2679:34399304] [AEP SDK DEBUG - <SignalHitProcessor>] Signal request successfully sent: https://adobe.com sent successfully
```

**Comment by emdobrin** [comment]
Looks like I had a typo in my example 😅 
I had multiple rules configured and I was not receiving the postbacks either, but that was because of an incorrect quote symbol `" vs ”`. 
Closing this as it works as expected
